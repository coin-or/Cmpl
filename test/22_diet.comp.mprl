<?xml version="1.0" encoding="UTF-8"?>
<mprl xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.coliop.org/schemas/Mprl.xsd">
	<general>
		<generalStatus>warning</generalStatus>
		<instanceName>22_diet.cmpl</instanceName>
		<mplName>CMPL</mplName>
		<message>cmpl finished with warnings</message>
	</general>
	<mplResult numberOfMessages="1">
		<mplMessage type ="warning" file="22_diet.cmpl" line="3" description="Use of deprecated assignment with '='"/>
	</mplResult>
</mprl>