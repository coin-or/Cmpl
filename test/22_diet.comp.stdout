---------------------------------------------------------------------------------------------------------
General status:     normal
Problem :           22_diet.cmpl
SolverName          GLPK
Nr. of Solutions:   1
---------------------------------------------------------------------------------------------------------
Solution nr.:       1
Objective name :    cost
Objective value :   101.14 (min!)

Variables           
Nr. of variables:   8
Name                 Type            Activity         Lower Bound         Upper Bound            Marginal
---------------------------------------------------------------------------------------------------------
x[BEEF]                 I                   2                   2                  10                   -
x[CHK]                  I                   8                   2                  10                   -
x[FISH]                 I                   2                   2                  10                   -
x[HAM]                  I                   2                   2                  10                   -
x[MCH]                  I                  10                   2                  10                   -
x[MTL]                  I                  10                   2                  10                   -
x[SPG]                  I                  10                   2                  10                   -
x[TUR]                  I                   2                   2                  10                   -
---------------------------------------------------------------------------------------------------------

Constraints         
Nr. of constraints: 4
Name                 Type            Activity         Lower Bound         Upper Bound            Marginal
---------------------------------------------------------------------------------------------------------
A                       G                1500                 700            Infinity                   -
B1                      G                1330                 700            Infinity                   -
B2                      G                 860                 700            Infinity                   -
C                       G                 700                 700            Infinity                   -
---------------------------------------------------------------------------------------------------------
